File: content-solutions/two-pointers/zeros-back.js
function solution(baskets) {
  const a = [...baskets];
  let w = 0;
  for (const x of baskets) {
    if (x !== 0) {
      a[w] = x;
      w++;
    }
  }
  a.fill(0, w);
  return a;
}

File: content-solutions/two-pointers/zeros-back.java
class Solution {
    public int[] solution(int[] baskets) {
        int[] answer = new int[baskets.length];
        int w = 0;
        for (int x : baskets) {
            if (x != 0) {
                answer[w] = x;
                w++;
            }
        }
        return answer;
    }
}
